### PR TITLE
Bug 1931232 - Add BMO vs Bugzilla warning block at top of new bug page for Bugzilla product

### PR DESCRIPTION
**Pull request by dklawren**
image

### CODE REVIEW

**Comment by dklawren** [comment]
> Personally, I don't love the way the warning is worded. I would specify that this is not the "component" for filing issues against BMO, since the website (ie the "place") is technically correct.
> 
> It also seems a little weird to list things that are incorrect for the current component and then refer to a different component. I'd change this to describe what the Bugzilla component is for ("this component tracks changes to the upstream Bugzilla project"), then say "you may be looking for the Mozilla-specific `BMO :: Admin` component, which is used for <list of config/permissions/account changes>".
> 
> Anyways I realize this is already in the code and you're just changing the conditions where it is revealed, so it works for me. 👍

I agree but didn't want to get too wordy as to take up a large part of the page with the warning. I will see if I can tweak the wording some and let you take a look.

**Review by cgsheeh — APPROVED** [review]
LGTM!
